doc/sphinx/source/n3fit/runcard_detailed.rst

```diff
@@ -346,6 +346,15 @@ should run by setting the environment variable ``CUDA_VISIBLE_DEVICES``
 to the right index (usually ``0, 1, 2``) or leaving it explicitly empty
 to avoid running on GPU: ``export CUDA_VISIBLE_DEVICES=""``
 
+Note that in order to run the replicas in parallel using the GPUs of a M1/M2 Mac, it is necessary to also install 
+the following packages:
+
+.. code-block:: bash
+
+   conda install -c apple tensorflow-deps
+   pip install tensorflow-macos==2.13.0 tensorflow-metal wandb==0.15.9
```

Comment:
Why is it necessary to really pin this version?

Reply:
It is related to this issue here: https://github.com/wandb/wandb/issues/5935

I was not able to make it run on MaC M2 GPUs with other versions.

Reply:
Could you also add a reference to that issue in the docs?